File: Entidades/Juego.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entidades
{
	public class Juego
	{
		[Key]
		public int IDJuego { get; set; }
        [RegularExpression(@".{3,}$", ErrorMessage = "El Nombre debe tener 3 letras minimo y no puede contener numeros o simbolos")]
  		[Required(ErrorMessage = "El Nombre es requerido")]
		public string? Nombre { get; set; }
        public int Precio { get; set; }
        public string? Imagen { get; set; }
        [NotMapped]
        public IFormFile? ImagenFile { get; set; }
        public int DevID { get; set; }
	}
}

File: Entidades/Juego.js
const { Sequelize, DataTypes, Model } = require('sequelize');
const { sequelize } = require('../db.js');

class Juego extends Model {}

Juego.init({
    IDJuego: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true
    },
    Nombre: {
      type: DataTypes.STRING,
      allowNull: true
    },
    Precio: {
      type: DataTypes.DOUBLE,
      allowNull: true
    },
    Imagen: {
      type: DataTypes.STRING,
      allowNull: true
    },
    DevID: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    Descripcion: {
      type: DataTypes.STRING,
      allowNull: true
    }
}, {

sequelize,
modelName: 'Juego',
tableName: 'juego',
timestamps: false
});

module.exports = Juego;